### ISSUE TITLE
Awesome project!

### ISSUE THREAD

**Issue by 0b01** [marechaux/gdax_rs#1]
Do you have plan to add websocket?

**Comment by marechaux** [comment]
Yes, my plan is to first handle all REST end points and document it, then, I'll start the web socket part.
